### PR TITLE
Disable flaky test

### PR DESCRIPTION
**Pull request by athei**
Unfortunately, the flakiness wasn't fixed by https://github.com/paritytech/polkadot-sdk/pull/3595. Let's disable the test in the meanwhile since it is hanging on the CI a lot.

### CODE REVIEW

File: substrate/client/consensus/manual-seal/src/lib.rs [hunk 1]
```diff
@@ -467,7 +467,9 @@ mod tests {
 		assert_eq!(client.header(created_block.hash).unwrap().unwrap().number, 1)
 	}
 
-	#[tokio::test]
+	// TODO: enable once the flakiness is fixed
```
Comment (liamaharon) [inline substrate/client/consensus/manual-seal/src/lib.rs:470]: Got a tracking issue to link to?

**Review by liamaharon — APPROVED** [review]

File: substrate/client/consensus/manual-seal/src/lib.rs (hunk 1 above)
Comment (athei) [reply substrate/client/consensus/manual-seal/src/lib.rs:470]: https://github.com/paritytech/polkadot-sdk/issues/3603

Comment (liamaharon) [inline substrate/client/consensus/manual-seal/src/lib.rs:470]:
```suggestion
	// TODO: enable once the flakiness is fixed
	// See <https://github.com/paritytech/polkadot-sdk/issues/3603>
```

**Review by liamaharon — APPROVED** [review]

**Comment by bkchr** [comment]
Instead of disabling the test, we should enable logging and see where it hangs/fails. Running this test locally doesn't lead to it stalling.

**Comment by athei** [comment]
I think @andresilva is debugging it. The fix can enable the test again.

This makes merging really hard. You need to be lucky two times in a row (you get kicked out of the merge queue).

**Review by andresilva — APPROVED** [review]

**Pull request merged**